Bottom: 339f90246a0eb12a69daeee1979c7001efb011ed
Top:    a795389078714322884023d7d2e4cda9654292d7
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-30 14:35:52 +0800

Refresh of tun-introduce-tx-array-mode

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index 3be69ea..7475215 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -1473,7 +1473,7 @@ static struct sk_buff *tun_ring_recv(struct tun_file *tfile, int noblock,
 		}
 
 		schedule();
-	};
+	}
 
 	current->state = TASK_RUNNING;
 	remove_wait_queue(&tfile->wq.wait, &wait);
